### PR TITLE
News on homepage

### PR DESCRIPTION
**Pull request by alanmoo**
Can I get some eyes on this? I had to make some best guesses re: design and content. See "View Deployment" link 3 items down from here

### CODE REVIEW

**Comment by mmmavis** [comment]
screen shot 2017-02-27 at 1 37 49 pm

Maybe use lighter font weight for titles on the right side? Should we also include `author` info for the post on the left? Maybe don't show Medium icon since other sources (e.g. blog.mozilla.org posts) don't have an icon attached to them?

**Comment by sabrinang** [comment]
On the left, I would bring the paragraph closer to the top (it is feeling separate and the spacing is too similar to the right side's in between space that separate news items). 

Also, I updated the icons to be a unified single colour grey for where the news to look more cohesive: https://drive.google.com/drive/folders/0B4Q8pzCpDS_eeUROaGsyVWhBcU0?usp=sharing

screen shot 2017-02-27 at 3 40 34 pm

File: pages/home.jsx [hunk 1]
```diff
@@ -108,12 +108,47 @@ var Home = React.createClass({
...
-          <p>Sign up for MozFest updates and be first in line to get your tickets for 2017.</p>
-          <a href="https://ti.to/Mozilla/mozfest-2017" className="btn btn-arrow"><span>Sign Up</span></a>
+        <div className="news content wide">
+          <h1 className="text-center">Latest News</h1>
+          <div className="row mt-5">
+            <div className="col-12 col-md-6 text-left">
+              <div className="news-widget mb-3">
+                <img src="assets/images/simone.png" alt="Simone Browne speaking on stage at MozFest" className="mb-2" style={{ "maxWidth": "100%" }} />
+                <div className="source medium-logo text-muted">Mozilla Festival</div>
+                <a href="https://medium.com/mozilla-festival/surveillance-and-race-online" className="title">Surveillance and Race Online</a>
```
Comment (mmmavis) [inline pages/home.jsx]: This url is `404`. Let's change it to https://medium.com/mozilla-festival/surveillance-and-race-online-c03dda350fb9 (ugly URL, I guest c03dda350fb9 is the revision number or something? But yea this is the post we want to link to)

**Comment by alanmoo** [comment]
@sabrinang Changing the medium logo color doesn't seem to fall within their [guidelines](https://medium.design/logos-and-brand-guidelines-f1a01a733592#.sfubb5nky), and I'm not sure where our new guidelines sit on that.

**Comment by sabrinang** [comment]
The medium guides suggest when it is less than ~50px it should only appear as a single solid color but it doesn't say it can't be in grey. l think this use case is okay and the medium logo is available on [font awesome](http://fontawesome.io/icon/medium/) that we can also make grey.

We have no solidified guidelines on our end but so far I'm unifying it in our grey so it looks cohesive system with our type and not distracting as a whole to serve as a visual indicator where the article is on and I know it is important to keep the integrity of other brands too.

File: public/index.html [hunk 2]
```diff
@@ -17,6 +17,7 @@
   <link rel="apple-touch-icon" type="image/png" sizes="152x152" href="/assets/images/favicon/apple-touch-icon-152x152@2x.png" />
   <link rel="stylesheet" type="text/css" href="/vendor/mofo-bootstrap/dest/css/mofo-bootstrap.css"/>
   <link rel="stylesheet" type="text/css" href="/index.css">
+  <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
```
Comment (mmmavis) [inline public/index.html]: should we just include `https://`?

**Comment by alanmoo** [comment]
@sabrinang this should be ready now

**Pull request merged**